extensions/adobe/experience/consumer-experienceevent.schema.json

```diff
@@ -0,0 +1,70 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/consumer-experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Consumer ExperienceEvent  Mixin",
+  "type": "object",
+  "description": "Analytics ExperienceEvent mixin for use with schemas for Solution data ingestion. Includes the core/standard ExperienceEvent as well as the required Adobe Analytics ExperienceEvent mixins.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/experienceevent-application",
+    "https://ns.adobe.com/xdm/context/experienceevent-channel",
+    "https://ns.adobe.com/xdm/context/experienceevent-environment-details",
+    "https://ns.adobe.com/xdm/context/experienceevent-marketing",
+    "https://ns.adobe.com/xdm/context/experienceevent-media",
+    "https://ns.adobe.com/xdm/context/experienceevent-search",
+    "https://ns.adobe.com/xdm/context/experienceevent-segmentmembership",
+    "https://ns.adobe.com/xdm/context/experienceevent-technical-details",
+    "https://ns.adobe.com/xdm/context/experienceevent-web",
+    "https://ns.adobe.com/experience/analytics/commerce"
+  ],
+  "definitions": {
+    "analytics-experienceevent": {
+      "properties": {}
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-application"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-channel"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-environment-details"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-marketing"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-media"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-search"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-segmentmembership"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-technical-details"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/experienceevent-web"
+    },
+    {
+      "$ref": "https://ns.adobe.com/experience/analytics/commerce"
```

Comment:
Should this be the standard commerce and not the analytics form?

Reply:
sorry my bad , i overlooked